test1.1/task1.1/task1.1.c

```diff
@@ -0,0 +1,94 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define ARRAY_SIZE 100
+
+bool isPalindrome(char* string, int stringSize)
+{
+    const int middle = stringSize / 2;
+    for (int i = 0; i <= middle; ++i)
+    {
+        if (string[i] != string[stringSize - i - 1])
+        {
+            return false;
+        }
+    }
+    return true;
+}
+
+void deleteSpaces(char* string)
+{
+    char* i = string;
+    char* j = string;
+    while (*j != 0)
```

Comment:
Наверное, имелось в виду 
```suggestion
    while (*j != '\0')
```